### PR TITLE
Added tests for Get all tournaments

### LINKED ISSUE #92: Get all tournaments must be tested
What has to be done:
Test the getAllTournaments functionality.

Why it has to be done:
So that we can be sure that the get all tournaments functionality is working properly

When the task is done:
When the functionality is tested with good and bad scenario's.

### PR DESCRIPTION
**Pull request by larsvandriel**
Closes #92

### CODE REVIEW

File: src/test/java/com/fontys/api/service/TournamentServiceTest.java [hunk 1]
```diff
@@ -1,33 +1,56 @@
 package com.fontys.api.service;
 
-import com.fontys.api.entities.Tournament;
-import com.fontys.api.entities.User;
-import com.fontys.api.repositories.TournamentRepository;
-import com.fontys.api.repositories.UserRepository;
-import org.junit.Before;
-import org.junit.Test;
-import org.mockito.Mockito;
```
Comment (chf3kl9) [inline src/test/java/com/fontys/api/service/TournamentServiceTest.java:9]: What happened here?

File: src/test/java/com/fontys/api/service/TournamentServiceTest.java [hunk 2]
```diff
@@ -109,4 +132,105 @@ private void createTournamentTestInvalid(String name, String description, Intege
...
+            fail();
+        }
+
+        assertEquals(expectedTournaments, actualTournaments);
+        verify(tournamentRepository, times(1)).findByOwner(user1);
+        verify(userRepository, times(1)).findById(user1.getId());
+    }
+
+    @Test
+    public void getAllTournamentsByOwnerIdNoTournaments()
```
Comment (chf3kl9) [inline src/test/java/com/fontys/api/service/TournamentServiceTest.java:262]: Isn't this an unnecessary test, as we already test getting all tournamentsByOwnerId?

File: src/test/java/com/fontys/api/service/TournamentServiceTest.java (hunk 1 above)
Comment (larsvandriel) [reply src/test/java/com/fontys/api/service/TournamentServiceTest.java:9]: I actually have no idea. But I fixed it now.

File: src/test/java/com/fontys/api/service/TournamentServiceTest.java (hunk 2 above)
Comment (larsvandriel) [reply src/test/java/com/fontys/api/service/TournamentServiceTest.java:262]: I think it's a nice way to test this unexpected scenario which is still valid. I think it's completely different from the tournamentsByOwnerId

**Review by chf3kl9 — APPROVED** [review]

**Review by lfb0801 — APPROVED** [review]
looks good

**Pull request merged**